cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -37,9 +37,36 @@ Examples:
 
   # Converts the CentOS image from the local filesystem with OS password set
   pvsadm image qcow2ova --image-name centos-82 --image-dist centos --os-password s0meC0mplexPassword --image-url /root/CentOS-8-GenericCloud-8.2.2004-20200611.2.ppc64le.qcow2
+
+  # Customize the image preparation script for RHEL/CentOS distro, e.g: add additional yum repository or packages, change name servers etc. 
+  # Step 1 - Dump the default image preparation template
+  pvsadm image qcow2ova --prep-template-default > image-prep.template
+  # Step 2 - Make the necessary changes the above generated template file(bash shell script) - image-prep.template
+  # Step 3 - Run the qcow2ova with the modified image preparation template
+  pvsadm image qcow2ova --image-name centos-82 --image-dist centos --image-url /root/CentOS-8-GenericCloud-8.2.2004-20200611.2.ppc64le.qcow2 --prep-template image-prep.template
 `,
 	PreRunE: func(cmd *cobra.Command, args []string) error {
 		opt := pkg.ImageCMDOptions
+
+		if opt.PrepTemplateDefault {
+			fmt.Println(prep.SetupTemplate)
```

Comment:
Just wondering if it would be more apt to just write to a default template file rather than print to a terminal and use redirection. Can there be any formatting issue that can crop up when printing to terminal ?

Reply:
I tried it on both mac and Linux machine, haven't seen any issues. I don't mind writing to the file, only thing is I didn't want to deal with any additional code to handle dumping content to a file. Let me know what do you think?

Reply:
Ok.. I'll leave it to you. I don't have any specific preference

Reply:
I will leave it as is as of now.